documentation/installation.md

```diff
@@ -166,6 +166,17 @@ kubectl logs <kyverno-pod-name> -n kyverno
 Here is a script that generates a self-signed CA, a TLS certificate-key pair, and the corresponding kubernetes secrets: [helper script](/scripts/generate-self-signed-cert-and-k8secrets.sh)
 
 
+## Install kyverno using helm 
```

Comment:
Can we move this to the top? Then add:

## Install Kyverno using YAMLs

And change the heading levels for "Option 1: ..." and "Option 2: ..." to level 3